sdk/storage/inc/datalake/response_models.hpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "blobs/internal/protocol/blob_rest_client.hpp"
+#include "protocol/datalake_rest_client.hpp"
```

Comment:
so `datalake_rest_client.hpp` isn't under `internal` directory anymore, we need to be consistent.

Reply:
As talked offline, this will not touch the code change and changes to CMake will remove the internal folder.